.github/workflows/package-helm.yaml

```diff
@@ -0,0 +1,35 @@
+name: Package Helm charts
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  CHARTS_DIR: deployment/helm/resource-management-policies/
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+
+      - name: Package Helm charts
+        run: |
+          helm package "$CHARTS_DIR"/*
```

Comment:
Should we rename the file so that it is more evident that this release asset is a Helm chart?

Reply:
sorry but which file do you mean?

Reply:
The files that `helm package` creates. I mean smth like
```
mv nri-resource-policy-balloons-v1.2.3.tgz nri-resource-policy-balloons-helm-chart-v1.2.3.tgz
```

So that the filename would give a hint that what tarball this actually is